<launch>
    <arg name="uavNumbers" default="8" />

    <node name="ground_control_station"
          pkg="ground_control_station"
          type="ground_control_station"
          output="screen">
        <param name="uavNumbers"  value="$(arg uavNumbers)" />
    </node>

    <include file="$(find ground_control_station)/launch/uav_visualization.launch"/>

</launch>